watertap/unit_models/electrolyzer.py

```diff
@@ -301,8 +300,8 @@ def custom_method_cathode(t, j):
         )
         self.anode_electrochem_potential = Var(
             initialize=1,
-            bounds=(0, None),
-            domain=NonNegativeReals,
+            bounds=(None, None),
```

Comment:
I will remove the bounds. Yes, in a full flowsheet, it would not solve as some of these values would be set to be negative even as they are fixed. Odd.